stage #1/module #1. Intro, Infrastructure & Instruments, Fundamentals/lesson #3. Classes, Intro to OOP/Weapon/src/test/java/com/epam/training/weapons/model/GunTest.java

```diff
@@ -0,0 +1,61 @@
+package com.epam.training.weapons.model;
+
+import org.junit.Test;
+import static org.junit.Assert.*;
+
+public class GunTest {
+
```

Comment:
same

Reply:
I will leave tests to gain 100% of coverage